module/zfs/zfs_sysfs.c

```diff
@@ -356,38 +356,68 @@ pool_property_show(struct kobject *kobj, struct attribute *attr, char *buf)
  * async calls (i.e. a sync flag). If this mechanism were in place at that
```

Comment:
Want to update this comment to use the real example or trim or initialize instead of the hypothetical?